src/server.cpp

```diff
@@ -826,21 +839,22 @@ static CTimingMeas JitterMeas ( 1000, "test2.dat" ); JitterMeas.Measure(); // TE
             }
         }
 
+        // use multithreading for any non-zero number of clients
+        // (overhead is low and it is worth doing for all numbers)
+        bUseMT = bUseMultithreading && iNumClients > 0;
+
         // prepare and decode connected channels
-        if ( !bUseMultithreading )
+        if ( !bUseMT )
         {
-            for ( int iChanCnt = 0; iChanCnt < iNumClients; iChanCnt++ )
-            {
-                DecodeReceiveData ( iChanCnt, iNumClients );
-            }
+            DecodeReceiveDataBlocks ( this, 0, iNumClients - 1, iNumClients );
```

Comment:
Is it worth a comment here to mirror the one at line 853, saying "run the OPUS decoder for all blocks" or whatever?